decidim-core/app/controllers/concerns/decidim/needs_participatory_process.rb

```diff
@@ -1,19 +1,28 @@
 # frozen_string_literal: true
 
-require "active_support/concern"
-
 module Decidim
   # This module, when injected into a controller, ensures there's a
   # Participatory Process available and deducts it from the context.
   module NeedsParticipatoryProcess
-    extend ActiveSupport::Concern
```

Comment:
Mmm partially. I think it kind of prevents people from learning ruby... But the thing here is, I couldn't use it because I needed the `extended` hook, which `ActiveSupport::Concern` doesn't provide.